package.json

```diff
@@ -64,6 +64,7 @@
     "lib"
   ],
   "dependencies": {
+    "moment": "^2.14.1",
```

Comment:
This is what is used for date formatting in force and metaphysics and it's super nice - any objections to its inclusion?